tests/test-extensions/some-extension/src/main/java/org/mule/test/some/extension/SomeExtension.java

```diff
@@ -7,22 +7,36 @@
 
 package org.mule.test.some.extension;
 
+import org.mule.runtime.extension.api.annotation.Configuration;
 import org.mule.runtime.extension.api.annotation.Export;
 import org.mule.runtime.extension.api.annotation.Extension;
 import org.mule.runtime.extension.api.annotation.Operations;
 import org.mule.runtime.extension.api.annotation.Sources;
 import org.mule.runtime.extension.api.annotation.connectivity.ConnectionProviders;
 import org.mule.runtime.extension.api.annotation.dsl.xml.Xml;
 import org.mule.runtime.extension.api.annotation.error.ErrorTypes;
+import org.mule.sdk.api.annotation.param.Optional;
+import org.mule.sdk.api.annotation.param.Parameter;
 import org.mule.test.heisenberg.extension.HeisenbergErrors;
 
+import java.time.ZonedDateTime;
+
 @Extension(name = "SomeExtension")
+@Configuration(name = "config")
 @ConnectionProviders(ExtConnProvider.class)
 @ErrorTypes(HeisenbergErrors.class)
-@Operations(SomeOps.class)
+@Operations({SomeOps.class})
 @Export(classes = CustomConnectionException.class)
-@Sources({SomeEmittingSource.class, AnotherEmittingSource.class, YetAnotherEmittingSource.class})
+@Sources({SomeEmittingSource.class, AnotherEmittingSource.class, YetAnotherEmittingSource.class,
+    ParameterEmittingSource.class})
 @Xml(namespace = "http://www.mulesoft.org/schema/mule/some", prefix = "some")
 public class SomeExtension {
 
+  @Parameter
+  @Optional
+  private ZonedDateTime zonedDateTime;
```

Comment:
Also if there aren't any differences in picking up a value from a config or a source or a parameter, maybe it is not needed to test all possible combinations (config + source + operation + another operation), but it does not hurt either... so I leave that up to you.